.github/workflows/ci.yaml

```diff
@@ -138,13 +138,47 @@ jobs:
         VERSION_PRERELEASE: dev
       run: make build
 
-    # Run e2e tests
+    - name: Save docker images
+      run: |
+        docker images --filter=reference='twingate/kubernetes-access-gateway:*' --format '{{.Repository}}:{{.Tag}}' | \
+          xargs -I {} sh -c 'docker save {} -o ./dist/$(echo {} | tr "/:" "-").tar'
```

Comment:
This would create a tar file for each Docker image? `docker save` supports multiple images in 1 tar archive  file, which should result simpler setup.

Reply:
Yeah, we should save multiple images into 1 tar archive file. (I didn't know `docker save` could do that 🙈 ) Updated in [d6a06a3](https://github.com/Twingate/kubernetes-access-gateway/pull/85/commits/d6a06a38fcd08cbe8aadfd5f8e14c6d7412103f5)